packages/components/modules/comments/CommentsList/index.tsx

```diff
@@ -13,7 +13,6 @@ import { CommentsListProps } from './types'
 
 const CommentsList: FC<CommentsListProps> = ({
   target: targetRef,
-  profileId,
```

Comment:
we should also remove the `profileId` from `CommentsListProps` and from its storybook